scripts/beaker/epwalsh-stability/llm-360-amber.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -exuo pipefail
+IFS=$'\n\t'
+
+BEAKER_LEADER_REPLICA_HOSTNAME=$1
+shift
+
+NUM_NODES=$1
+shift
+
+BEAKER_REPLICA_RANK=$1
+shift
+
+# Setup Python environment.
+conda shell.bash activate base
+
+# Install flash-attn
```

Comment:
I'm not sure if this is from your stability experiments. I haven't been doing this... not sure if I should be?

Reply:
You might already have flash in your image.